src/components/common/animations/Feed/Base.js

```diff
@@ -10,11 +11,21 @@ class FeedInfo extends React.Component {
   }
 
   componentDidMount() {
-    const { delay = 0 } = this.props
+    const { delay = 0, asImage } = this.props
 
     this.anim.onComplete = this.onAnimationFinishHandler
 
-    setTimeout(() => this.anim.play(), delay)
+    if (asImage) {
+      const lastFrame = Number(this.animationData.op) - 1
```

Comment:
as i remember, math operators (except "+" as it could mean also strings concat)  should automatically convert variable to number

Reply:
Yep:
<img width="116" alt="image" src="https://user-images.githubusercontent.com/62208067/77337335-6c898b80-6d31-11ea-8661-d95552132f77.png">

Reply:
<img width="92" alt="image" src="https://user-images.githubusercontent.com/62208067/77337377-7b703e00-6d31-11ea-802c-a1279dd8cb82.png">

Reply:
I did exactly like you saying - but prettier changed + with Number before commit

Reply:
ok